src/components/Filter/components/SearchInput/SearchInput.style.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from '@emotion/core';
+
+import { Theme } from 'theme';
+
+export const textFieldWrapper = () => (theme: Theme) => css`
+  border-bottom: 1px solid ${theme.utils.getColor('lightCoolGray', 200)};
+  
+  div:first-of-type {
```

Comment:
Is there not any variation of the TextField component full filling our needs here? 

It's small overrides that are happening but I would prefer to use things unchanged. This is more of a design question. 

Adding @panvourtsis to the discussion.

Reply:
I did not find any prop/way in TextField component to do this without adding custom css.

I will confirm with the design team again and let you know

Reply:
@Christos-Zacharopoulos yes just confirmed with the design team...we don' t want here the behavior of the `TextInput`.